csibe.py

```diff
@@ -108,6 +110,38 @@ def submodule_init_and_update(repository_path):
         return
 
 
+def download_old_testbed():
+    old_csibe_tar_filename = "CSiBE-v2.1.1.tar.gz"
```

Comment:
Add a separate variable for the version, and use it later. Probably it should be global (see below).